.gitignore

```diff
@@ -5,3 +5,5 @@ venv/
 .idea/
 .DS_Store
 .Rproj.user
+apps/dashr-ncbi-explorer/restructuredApp.R
```

Comment:
I would move these files to a gitignore specific to the app.

Reply:
Done.